sales/send_quotations/get_paid_to_validate.rst

```diff
@@ -2,42 +2,56 @@
 Get paid to confirm an order
 ============================
 
-You can use online payments to get orders automatically confirmed.
-Saving the time of both your customers and yourself.
+In general, online payments are considered as a fast and secure alternative to traditional payment
+methods. It is generally cheaper, easier, faster and highly efficient for international transactions
+than other payment methods. With Odoo Sales, you can use online payments to get automatic order
```

Comment:
Doesn't work. 

Would switch to "It is generally cheaper, easier and faster than other payment methods. It is particularly useful and efficient for international transactions."